ros_buildfarm/templates/snippet/scm_git.xml.em

```diff
@@ -6,6 +6,9 @@
         <refspec>@ESCAPE(refspec)</refspec>
 @[end if]@
         <url>@ESCAPE(url)</url>
+@[if vars().get('git_ssh_credential_id')]@
```

Comment:
Not all uses of this snippet set a git_ssh_credential_id, so I wanted to avoid an error in the template expansion due to an unset variable.